blog/2025-08-06-phasetwo-cockroachdb-partnership.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Phase Two Recognized as Official CockroachDB Partner
+slug: phasetwo-cockroachdb-partnership
+date: 2025-08-06
+authors: phasetwo
+tags: [phase_two, keycloak, cockroachdb, partnership, database, performance]
+---
+
+<div
+  style={{ display: "flex", alignItems: "center", gap: 10, marginBottom: 20 }}
+>
+  <img
+    src="/blog/crdb_partnership/crdb.webp"
+    alt="CockroachDB logo"
+    style={{ boxShadow: "none" }}
+  />
+  <span>+</span>
+  <img
+    src="/img/logo-phasetwo.png"
+    style={{ boxShadow: "none" }}
+    alt="Phase Two Logo"
+  />
+</div>
+
+[Phase Two](https://phasetwo.io) is thrilled to announce that we have been recognized as an **official partner** of [CockroachDB](https://www.cockroachlabs.com/partners/locator/). This partnership marks a significant milestone in our commitment to providing robust, scalable, and high-performance database solutions for our the Keycloak community and our customers.
+
+Phase Two originally built our [CockroachDB integration](https://github.com/p2-inc?q=crdb&type=all&language=&sort=#our-docker-images) for [Keycloak](https://quay.io/repository/phasetwo/keycloak-crdb) over two years ago, and since then we have been working closely with the CockroachDB team to ensure that our integration is optimized for performance and reliability. Our customers have seen significant improvements in database performance, scalability, and a overall cost savings by using CockroachDB with Keycloak.
+
+We believe in it that all of our [hosting](https://phasetwo.io/hosting) is built using CockroachDB.
```

Comment:
Not sure of the language here. What about?

Our commitment to CockroachDB is built on years of using the [Cockroach Cloud](https://www.cockroachlabs.com/product/cloud/) to power all of our dedicated [hosting](https://phasetwo.io/hosting).